frontend/src/Series/Index/Overview/SeriesIndexOverviewInfo.js

```diff
@@ -182,6 +183,7 @@ function SeriesIndexOverviewInfo(props) {
     nextAiring,
     showRelativeDates,
     shortDateFormat,
+	longDateFormat,
```

Comment:
this line should be removed too, i think....

Reply:
Yes, it will fail the linting.